Source/Core/DolphinWX/AboutDolphin.cpp

```diff
@@ -29,51 +31,94 @@ AboutDolphin::AboutDolphin(wxWindow *parent, wxWindowID id,
 	wxStaticBitmap* const sbDolphinLogo = new wxStaticBitmap(this, wxID_ANY,
```

Comment:
This should actually be a wxGenericStaticBitmap, and is recommended in wx documentation for large images. You can get access to that by changing `#include <wx/statbmp.h> to `#include <wx/generic/statbmpg.h>` (don't forget to move it after the windowid.h include so you don't trip the lint check).